snappy_pipeline/workflows/adapter_trimming/__init__.py

```diff
@@ -0,0 +1,334 @@
+# -*- coding: utf-8 -*-
+"""Implementation of the ``adapter_trimming`` step
+
+The adapter_trimming step performs adapter & quality trimming of reads (DNA or RNA).
+The tools are highly configurable, and provide feedback of the success of the operation.
+
+==========
+Step Input
+==========
+
+Adapter trimming starts at the raw RNA-seq reads.  Thus, the input is very similar to one of
```

Comment:
Probably better to just describe the input even if you are just copying and pasting the information. You already reference `ngs_mapping` below.